### DISCUSSION TITLE
Streaming

### DISCUSSION THREAD

**Discussion by ryanjohn80** [dgreif/ring#1442] [General]
Hi all, I have the latest version of the plugin but in HomeKit it only streams the first 3 seconds of video, then the video freezes and stays frozen on that frame but the audio continues to work perfectly.  I checked the streaming FAQ and didn't see anything in there.  Any ideas?  Thanks vm

**Comment by tsightler** [comment]
This usually indicates some type of networking related issue.  Homebridge-ring basically just forwards RTP video packets from Ring unmodified.  The initial UDP traffic from Ring cameras can be quite bursty and packets might be dropped which can cause video artifacts or freezing as Homekit is not particularly robust against such issues.

**Comment by ryanjohn80** [comment]
Makes sense. Appreciate the response as always my man.  On Jun 30, 2024, at 12:15 PM, tsightler ***@***.***> wrote:﻿
This usually indicates some type of networking related issue.  Homebridge-ring basically just forwards RTP video packets from Ring unmodified.  The initial UDP traffic from Ring cameras can be quite bursty and packets might be dropped which can cause video artifacts or freezing as Homekit is not particularly robust against such issues.
